samples/client-sdk/knocking/index.js

```diff
@@ -146,9 +146,9 @@ const addParticipantVideo = async (participant) => {
   }
 };
 
-const checkAccessLevel = async () => {
+const checkAccessLevel = () => {
   // https://docs.daily.co/reference/daily-js/instance-methods/access-state
-  const state = await callObject.accessState();
+  const state = callObject.accessState();
```

Comment:
After checking our type def and then the implementation as well (in case the type def is wrong), I'm pretty sure this is a synchronous method